conan/cli/cli.py

```diff
@@ -190,6 +190,7 @@ def run(self, *args):
 
         try:
             command.run(self._conan_api, args[0][1:])
+            _warn_frozen_center(self._conan_api)
```

Comment:
Why not adding this check in the ``remotes.py`` subapi? seems more correct there, at remotes loading file, without necessarily doing an extra load here.

Reply:
That was also my first approach but then the warning is shown at the beginning of the operation and gets diluted between all the other output and I decided to bring it here so that's always the last thing the user will see.